### DISCUSSION TITLE
First steps...can't read created OIDs...

### DISCUSSION THREAD

**Discussion by Micharius** [entropia/tip-toi-reveng#269] [Q&A]
Hi there

I tried to do first steps with ttool, I might need some help: I carefully read all the Wiki articles and created a first test book for my daughter's Tiptoi pen, including the PNG and the oid-table as PDF file. 
As a test, I printed out the [sample page from Ravensburger.](https://www.ravensburger.de/content/wcm/mediadata/pdf/Entdecken/Ravensburger%20Marken/tiptoi/tiptoicode_Navigationsleiste.pdf) Even with my HP ENVY Photo 7800, I was able to  read them back with the Tiptoi pen. "No big deal" I tought;, and  so I printed out the OID codes created with tttool. Unfortunately, it wasn't possible to read them with the pen. I took the files in my office, where I did countless tests with the different laserprinters we have there. 1200dpi, very good quality, only grayscale / black white etc., But still, I'm not able to read back the various printed code. I wasn't shure if I made a mistake while creating the YAML, so I used one of the GME files from the pen to disassemble, with no success, too. 
Did I miss a part in the process? Are the generated OID codes indeed very delicate to handle and what is the tolerance in quality? I'm looking forward for any tipps and tricks.

Kind regards,

Micharius

**Comment by nomeata** [comment]
Printing is indeed delicate, but if the sample page works, you can be optimistic.

If you compare the print-outs visually (maybe using a magnifying glass), any obivious difference? Scale, crispness?

**Comment by Micharius** [comment]
Hello, I did some tests today - all with my HP Envy Photo 7830, see attached PDF, I made some photo from the print results with my camera. The "original" Ravensburger it definitely better in quality and much sharper. Althoug, both PDFs, compared in Acrobat Reader with 4000% scale, look identical.
Somehow, the Ravensburger PDF must contain more or different informations for the printer, I guess.
What do you guys use to process the OID codes generated from tttool? The PDF, SVG or PNG, and which program do you use to further process the files? 
[print comparison.pdf](https://github.com/entropia/tip-toi-reveng/files/7921361/print.comparison.pdf)

**Comment by mrbryce** [comment]
gimp or photoshop
RGB or grayscale only (no CMYK)
edit in 1200ppi ONLY

1200dpi laser printer or 2400dpi inkjet printer

print with default settings and two seperate coatings
one page with colors
the other page with the OID

use PNG oids without transparency

**Reply by mariusa** [reply]
Hi @mrbryce ,

> the other page with the OID

You mean, put again the printed page with artwork back in the printer and print again on it the 2nd image only with OIDs ?
